python/jupyterlab-collaborative-chat/schema/factory.json

```diff
@@ -37,6 +37,12 @@
       "default": true,
       "readOnly": false
     },
+    "defaultDirectory": {
+      "description": "Default directory where to create and look for chat.",
+      "type": "string",
+      "default": "",
```

Comment:
Should the setting provide more information (for example via the `description`) on where files are saved by default? Since an empty string can be ambiguous?